BlogTemplate/Pages/Drafts.cshtml

```diff
@@ -2,20 +2,28 @@
 @model DraftsModel
 @{
     ViewData["Title"] = "Drafts";
+
     <h2>@ViewData["Title"]</h2>
 
-    if (!Model.Posts.Any())
+    if (!Model.DraftSummaries.Any())
     {
         <p>View and edit your saved drafts here.</p>
     }
 
     else
     {
-        foreach (var post in Model.Posts)
+        foreach (var post in Model.DraftSummaries)
         {
             <article>
-                <h2><a href="/edit/@post.Id/@post.Slug">@post.Title</a></h2>
-                <p class="lead">@post.Excerpt</p>
+                <div class="col-md-12">
+                    <h2><a href="/edit/@post.Id/@post.Slug">@post.Title</a></h2>
+                    @if (post.PublishTime != null)

```

Comment:
DateTimeOffset cannot be null because it is a struct.  Instead, you should compare with `default(DateTimeOffset)`